src/components/MainMenuItem.js

```diff
@@ -19,7 +19,7 @@ const MainMenuItem = ({ component, isMobileSized, id, ...props }) => {
 };
 
 MainMenuItem.propTypes = {
-  component: PropTypes.node,
+  component: PropTypes.func,
```

Comment:
Is it always a func ?

Reply:
in fact, this is always a component, but console warns with de `propTypes.node`. Maybe there is a bette, explicite, propTypes for component?

Reply:
@jrmi maybe i could set 

```suggestion
  component: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
```

If i remove func, in some cases there is a warn.